Services/CO.CDP.Organisation.WebApi/Properties/launchSettings.json

```diff
@@ -14,7 +14,7 @@
       "dotnetRunMessages": true,
       "launchBrowser": true,
       "launchUrl": "swagger",
-      "applicationUrl": "http://localhost:8082",
+      "applicationUrl": "http://localhost:7208",
```

Comment:
Any reason for this change? Some time ago I aligned these ports with docker. Isn't that what we want?

Reply:
Make sense, will revert change